libraries/bdp-access-proxy/bdp-access-impl/src/main/java/com/bosch/osmi/bdp/access/impl/model/LicenseImpl.java

```diff
@@ -60,7 +60,7 @@ public boolean hasLicenseTextAttached(){
         } catch (SdkFault sdkFault) {
             LOGGER.error("Unable to get license text from Bdp server. Reason: \n" + sdkFault.getMessage());
             LOGGER.debug("License ID: " + getId());
-            throw new IllegalStateException(sdkFault.getCause());
+            return false;
```

Comment:
I'm still a little bit confused as to why UserImpl, ProjectInfoImpl, and ProjectImpl throw on SdkFault, but LicenseImpl and ComponentImpl log an SdkFault and return some defaults. So it's consistent within each class, but inconsistent across the package. I'm not implying that's a problem, but I can't help but notice it.